plugin/trino-faker/src/main/java/io/trino/plugin/faker/FakerFunctions.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.faker;
+
+import io.airlift.slice.Slice;
+import io.trino.spi.function.Description;
+import io.trino.spi.function.ScalarFunction;
+import io.trino.spi.function.SqlType;
+import net.datafaker.Faker;
+
+import static io.airlift.slice.Slices.utf8Slice;
+import static io.trino.spi.type.StandardTypes.VARCHAR;
+
+public final class FakerFunctions
+{
+    private final Faker faker;
+
+    public FakerFunctions()
+    {
+        faker = new Faker();
+    }
+
+    @ScalarFunction
+    @Description("Generate a random string based on the Faker expression")
+    @SqlType(VARCHAR)
+    public Slice randomExpression(@SqlType(VARCHAR) Slice expression)
```

Comment:
I disagree, it would suggest the function returns a faker expression, not a random string based on a faker expression. I agree the current name is not the best, but wanted to keep the function name short.

Reply:
fakerRandom ?
Something in the name needs to indicate that we expect a faker expression as input, e.g. https://trino.io/docs/current/functions/regexp.html

Reply:
How about `random_string` with a `faker_expression` argument?

Reply:
I can also drop this function entirely to avoid blocking the PR, but it's useful for testing Faker expressions to be used in column properties.

Reply:
generateFromExpression ?